assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/AnalyticsStats.js

```diff
@@ -85,6 +88,12 @@ const AnalyticsStats = ( {
 		},
 	};
 
+	if ( isZeroReport( data ) ) {
+		options.vAxis.viewWindow.max = 1;
+	} else {
+		options.vAxis.viewWindow.max = undefined;
+	}
+
```

Comment:
This is the same approach used elsewhere; if this isn't used then it's possible the chart heights get messed up, I think because of either loading states but possibly/also because of how the Chart component works. The IB omitted this but it's needed.

I also opted to use the `isZeroReport` function rather than the custom function in the IB, because this worked fine to identify a non-zero report.

Reply:
Unfortunately we can't rely on `isZeroReport` due to reasons mentioned above. I think for these two, we can simply look at the respective index (`selectedStats`) of each item in `data.0.data.totals`.